python/pyspark/pandas/frame.py

```diff
@@ -10081,6 +10086,10 @@ def all(self, axis: Axis = 0) -> "Series":
             * 0 / 'index' : reduce the index, return a Series whose index is the
               original column labels.
 
+        bool_only : bool, default None
+            Include only boolean columns. If None, will attempt to use everything,
+            then use only boolean data. Not implemented for Series.
```

Comment:
`Not implemented for Series.` I guess this useless, because `frame.all` is not in `generic` like pandas.

Reply:
Makes sense! Removed.